server/src/main/java/io/unitycatalog/server/service/credential/CredentialOperations.java

```diff
@@ -41,9 +42,13 @@ public TemporaryCredentials vendCredential(String path, Set<CredentialContext.Pr
   }
 
   public TemporaryCredentials vendCredential(CredentialContext context) {
+    var location = context.getLocations().get(0);
```

Comment:
nit: i dont think we use `var` in our codebase. so lets use actual class names here.

Reply:
Since we're on Java 17 and so Scala-ish I thought it'd be quite nice (productive?) to use it. I'm fine with a type annotation 🤷

Reply:
In scala we rarely use var, always val if nothing needs to be updated. So var seems inconsistent with our usual scala-style as well as rest of the java code style in the project. 
not a very strong opinion, but its nice to have consistent coding style throughout a project.

Reply:
Nah, I said "Scala-ish" to mean "avoid types" and the only way to do so in Java is `var`.

I fully agree with you on `var` in Scala.